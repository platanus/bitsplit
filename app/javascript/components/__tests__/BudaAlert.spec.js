import { shallowMount } from '@vue/test-utils'
import BudaAlert from '../BudaAlert'

describe('BudaAlert', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(BudaAlert, {
      stubs: ['router-link']
    })
    expect(wrapper).toMatchSnapshot()
  })
})
