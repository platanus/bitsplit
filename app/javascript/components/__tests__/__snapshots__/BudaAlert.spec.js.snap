// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BudaAlert matches snapshot 1`] = `
<div role="alert" class="bg-teal-100 border-t-4 border-teal-500 rounded-b text-teal-900 px-4 py-3 shadow-md">
  <div class="flex">
    <div class="py-1"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" class="fill-current h-6 w-6 text-teal-500 mr-4">
        <path d="M2.93 17.07A10 10 0 1 1 17.07 2.93 10 10 0 0 1 2.93 17.07zm12.73-1.41A8 8 0 1 0 4.34 4.34a8 8 0 0 0 11.32 11.32zM9 11V9h2v6H9v-4zm0-6h2v2H9V5z"></path>
      </svg></div>
    <div>
      <router-link-stub to="buda">
        <p class="font-bold">No est√°s conectado a Buda</p>
      </router-link-stub>
      <p class="text-sm">Para realizar pagos debes asociar una cuenta Buda.</p>
    </div>
  </div>
</div>
`;
